src/features/abi-methods/components/decoded-abi-method-arguments.tsx

```diff
@@ -1,72 +1,82 @@
 import { useMemo } from 'react'
-import { AbiMethod, AbiType } from '@/features/abi-methods/models'
+import { AbiMethodArgumentRepresentation, AbiType } from '@/features/abi-methods/models'
 import { AbiValue } from '@/features/abi-methods/components/abi-value'
 import { TransactionLink } from '@/features/transactions/components/transaction-link'
 import { AccountLink } from '@/features/accounts/components/account-link'
 import { ApplicationLink } from '@/features/applications/components/application-link'
 import { AssetIdLink } from '@/features/assets/components/asset-link'
 
-export function DecodedAbiMethodArguments({ method }: { method: AbiMethod }) {
+export function DecodedAbiMethodArguments({
+  arguments: argumentsProp,
+  multiLine,
+}: {
+  arguments: AbiMethodArgumentRepresentation[]
+  multiLine: boolean
+}) {
+  const renderArgumentValue = (argument: AbiMethodArgumentRepresentation) => {
+    if (argument.type === AbiType.Transaction) {
+      return (
+        <TransactionLink className="text-primary underline" transactionId={argument.value}>
+          {argument.value}
+        </TransactionLink>
```

Comment:
```suggestion
        <TransactionLink transactionId={argument.value} />
```
same for other links.